src/selective.d

```diff
@@ -33,13 +34,17 @@ final class SelectiveSync
 		// Does the file match skip_file config entry?
 		// Returns true if the file matches a skip_file config entry
 		// Returns false if no match
+		log.vlog("isNameExcluded for name '", name, "': ", !name.matchFirst(mask).empty);

```

Comment:
I like the idea of logging if something is matched, however at a verbose level this becomes too much - potentially better suited to a --debug flag option at runtime